cmd-record.c

```diff
@@ -245,42 +311,25 @@ static void setup_child_environ(struct opts *opts, int pfd)
 	if (opts->patt_type != PATT_REGEX)
 		setenv("UFTRACE_PATTERN", get_filter_pattern(opts->patt_type), 1);
 
-	if (opts->lib_path)
-		snprintf(buf, sizeof(buf), "%s/libmcount/", opts->lib_path);
-	else
-		buf[0] = '\0';  /* to make strcat() work */
+	libpath = get_libmcount_path(opts);
+	if (libpath == NULL)
+		pr_err("cannot found libmcount.so");
```

Comment:
Please use `pr_err_ns` instead since we don't know the error code at the time.

Reply:
Yep !